### DISCUSSION TITLE
Firefox

### DISCUSSION THREAD

**Discussion by Marco-9456** [nextai-translator/nextai-translator#64] [Q&A] [answered]
How can I install it in Firefox?

**Comment by maltoze** [comment, marked as answer]
currently, you can install it in firefox by following the steps below:
1. run `yarn && yarn build`
2. go to `about:debugging#/runtime/this-firefox` on firefox
3. click `Load Temporary Add-on...`
4. select `build/firefox/manifest.json` in project path
